CMakeLists.txt

```diff
@@ -134,6 +134,7 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Linux" OR CMAKE_SYSTEM_NAME STREQUAL "Android")
         "-I/usr/lib/swift") # dispatch
 endif()
 
+include(GNUInstallDirs)
```

Comment:
Just double check: it's okay to set it for all platforms (not just windows right?)

Reply:
Yep! Linux tests + toolchain builds pass with this change too - I suspect this would have also led to an issue on Linux with `plutil` not being installed since it too would be installed into the runtime directory, so it's likely not only safe but necessary there too